app/src/test/report/reducers/ReportsReducersTests.ts

```diff
@@ -28,4 +28,15 @@ describe('Reports reducer tests', () => {
         expect(newState.versionDetails).to.eql(versionMock);
     });
 
+    it('sorts reports list', () => {
+        const reportsMock = [mockReport({name: "b"}), mockReport({name: "c"}), mockReport({name: "a"})];
```

Comment:
This tests that they are sorted by name, just need another test to make sure they are sorted by date as expected